hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassDefinitionsServiceImpl.java

```diff
@@ -1909,11 +1928,15 @@ private Dampening mapDampening(Row row) {
                     throw new IllegalArgumentException("Unexpected ConditionType: " + cond);
                 }
 
+                // TODO: ARE THESE TAGS STILL NEEDED BY THE HAWKULAR UI? THIS IS CLUNKY.
+                // I DON'T THINK THEY ARE USED. REMOVING...
```

Comment:
Thanks for confirming. I looked in the UI code and could not find any code using these built-in tags, which were originally added because the UI needed it.  But I think it's no longer the case and I want to remove them.